### PR TITLE
SBERDOMA-498 Mobile phone captcha check

### PR DESCRIPTION
**Pull request by dkoviazin**
Add captcha check for mobile devices

### CODE REVIEW

File: apps/condo/domains/user/utils/googleRecaptcha3.js [hunk 1]
```diff
@@ -4,39 +4,56 @@ const captchaConfig = conf.GOOGLE_RECAPTCHA_CONFIG ? JSON.parse(conf.GOOGLE_RECA
...
     }
+    if (mobilePlatform) {
+        try {
+            let result = false
+            eval(mobileCapthaCheck)
+            if (result) {
+                return { error: null }
+            }
+        } catch (error) {
+            console.warn('[error] mobile captcha check failed', error)
```
Comment (toplenboren) [inline apps/condo/domains/user/utils/googleRecaptcha3.js:38]: I find this message ambigious, we say error, but call warn

Comment (dkoviazin) [reply apps/condo/domains/user/utils/googleRecaptcha3.js:38]: It's debugging log to check mobile integration. I will need to trace requests. 
The real error will be thrown on next lines

File: apps/condo/domains/user/schema/ConfirmPhoneAction.js [hunk 2]
```diff
@@ -63,6 +63,23 @@ const checkLock = async (phone, action) => {
...
+        if (isIpad || isIPhone || isIPod) {
+            return 'iOS'
+        }
+        if (isAndroid) {
+            return 'Android'
+        }
+    }
+    return ''
+}
+
```
Comment (AntonAL) [inline apps/condo/domains/user/schema/ConfirmPhoneAction.js:82]: Why not to use third-party libraries to parsing userAgent string, like [ua-parser-js](https://www.npmjs.com/package/ua-parser-js) or [bowser](https://www.npmjs.com/package/bowser)?

File: apps/condo/domains/user/utils/googleRecaptcha3.js [hunk 3]
```diff
@@ -4,39 +4,56 @@ const captchaConfig = conf.GOOGLE_RECAPTCHA_CONFIG ? JSON.parse(conf.GOOGLE_RECA
...
 
-const captchaCheck = async (response, action = '') => {
+const captchaCheck = async (response, action = '', mobilePlatform = '') => {
     if (conf.NODE_ENV === 'test') {
         return { error: null }
     }
+    if (mobilePlatform) {
+        try {
+            let result = false
+            eval(mobileCapthaCheck)
```
Comment (pahaz) [inline apps/condo/domains/user/utils/googleRecaptcha3.js:33]: it's a bad idea to eval data from ENV

**Pull request closed without merging**